pos_backend_partner/static/src/js/pos_partner.js

```diff
@@ -0,0 +1,77 @@
+/* Copyright (C) 2020-Today Akretion (https://www.akretion.com)
+    @author Raphaël Reverdy (https://www.akretion.com)
+    License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+*/
+
+odoo.define("pos_backend_partner.ProductScreen", function (require) {
+    "use strict";
+    var Registries = require("point_of_sale.Registries");
+    var ProductScreen = require("point_of_sale.ProductScreen");
+    var PaymentScreen = require("point_of_sale.PaymentScreen");
+    var tools = require("pos_backend_communication.tools");
+
+    var pos_instance = undefined;
```

Comment:
pos_instance and _t seem to raise en issue for pylint https://github.com/OCA/pos/runs/6715157129?check_suite_focus=true#step:7:85

useless error ?